README.md

```diff
@@ -170,7 +174,7 @@ Example:
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- **Add more Classes**
+- **REST Full API**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] The MIT license link in your readme file is displaying a 404 error. Kindly fix the link to your license file.